### DISCUSSION TITLE
sandbox card

### DISCUSSION THREAD

**Discussion by kalebbroo** [krishsharma0413/DiscordLevelingCard#13] [Q&A] [answered]
Is there a way to not include things that are normally required like avatar and level ? Im thinking i want to make some welcome cards and a leaderboard card but dont see a native way to do that. If i made that would you want that in a pr?

**Comment by krishsharma0413** [comment, marked as answer]
hmmmm
Even though this library was made with the purpose of only adding level cards but using it for leaderboard and welcome doesn't sound too bad. I am busy cause of my exams and it would be great if you help me to add it yourself! 

Adding a new method in sandbox class which gives you access to adding any number of images(we can use it for avatars) and removing unneeded options would be great.

**Comment by kalebbroo** [comment]
Still working on this but here is a proof of concept.

![image](https://github.com/krishsharma0413/DiscordLevelingCard/assets/10682641/c3a03622-2c5a-466e-ada6-451819440779)

**Reply by krishsharma0413** [reply]
that doesn't look like a good idea

**Reply by krishsharma0413** [reply]
i mean, the idea is good but the design doesn't seem ideal for a lot of usecases

**Comment by krishsharma0413** [comment]
![image](https://github.com/krishsharma0413/DiscordLevelingCard/assets/77439837/1cfc0bef-c3fa-4711-a6e5-b8efa575607d)

I have created a layout similar to this previously and i think something like this would be ideal design.
It has aesthetic look and provides everything needed

**Reply by kalebbroo** [reply]
Haha yeah that is just a proof of concept that my code works. Not an actual design at all.

**Comment by kalebbroo** [comment]
I am not sure how I would add all of the avatar images for each user without completely rewriting the sandbox. If you point me in the right direction I think I can do most of the work. I am still a beginner so my code is not great. I am taking a trip to Japan on the 9th so I will have like 14 hours on a plane. I plan to write a bunch of code.

**Comment by krishsharma0413** [comment]
You can definitely trying making two methods in a new class called `Leaderboard` where you can add the two methods. One method would not include avatars which would be significantly easy. Another method could use a `list` to store data of each user such as `[username, avatar, level, exp, rank(?)]` instead of adding this feature to `Sandbox` i think making it a completely new feature would be more suited for this.
